src/Extensions/Abstractions/Activities/ActivityExtensions.cs

```diff
@@ -37,6 +37,24 @@ public static class ActivityExtensions
 				HealthCheckMarkerValue,
 				StringComparison.OrdinalIgnoreCase);
 
+		/// <summary>
+		/// Returns true if activity is marked as Performance test
+		/// </summary>
+		/// <param name="activity"></param>
+		/// <returns></returns>
+		public static bool IsPerformanceTest(this Activity activity) =>
+			!string.IsNullOrEmpty(activity.GetBaggageItem(PerformanceMarkerKey));
+
+		/// <summary>
+		/// Returns the value of the Performance key if it exists
+		/// </summary>
+		/// <param name="activity"></param>
+		/// <returns></returns>
+		public static string PerformanceValue(this Activity activity)
```

Comment:
Why this should be a separate method and why it's public, would marker value have some significance or it's just a flag?

Reply:
The idea for the performance marker is that an int value can be passed instead of just a boolean value (true or false).  This int value could be used to mock a response time for a partner call, e.g. PerfomanceMaker=1000 would mock a 1000 millisecond response from a partner service to see how the calling service would handle such a delay.

Reply:
I'm not sure that I fully understand the idea. 
1. If this value is intended to be used as configuration settings only for mocks that would emulate delay from external service, why could not this configuration be set in the mock itself without transferring additional parameters?
2. What would be the benefit of having this functionality in shared code? Would it be needed for many services or some specific use case? We would probably need more description and explanation of how other people could use it. I do not fully understand what mocks are we talking about. Maybe you could point to some sample usage, please?
3. If we are going forward with this approach it might be better for `PerformanceValue` method to return `int` or `int?` instead since every consumer would need to parse it. And then we can have a corresponding set method to avoid duplicating string key (and risk of them having different values) in the place where you planning to set the value. And some simple UT would be nice.

@martinpe36  @jj05y  would you like to set up a short call to discuss the best way of doing it?